src/Nest/Search/SearchTemplate/PutSearchTemplate/ElasticClient-PutSearchTemplate.cs

```diff
@@ -7,37 +7,57 @@ namespace Nest
 {
 	public partial interface IElasticClient
 	{
-		/// <inheritdoc/>
+		/// <summary>
+		/// Stores a search template that can be used to pre render search requests,
+		/// before they are executed and fill the search template with template parameters.
+		/// </summary>
+		[Obsolete("Removed in NEST 6.x. In NEST 6.x, use the PutScript API to store templates")]
```

Comment:
Removed in NEST 6.x, use the PutScript API to store templates

Reply:
A few instances that have this phrasing.

Reply:
Users should continue to use `PutSearchTemplate` in 5.x, but when moving to 6.x, should use `PutScript`. I thought it was ambiguous when `In NEST 6.x, ` is left out as to whether users should transition to `PutScript` _now_ or when they move to 6.x, so thought it was clearer to indicate when users should move to using `PutScript`

Reply:
Is that clear? 😄